app/api/user_check_in.py

```diff
@@ -77,31 +76,16 @@ def before_post(_args, _kwargs, data):
         :return:
         """
         require_relationship(['station'], data)
-        if not has_access('is_coorganizer', station=data.get('station')):
```

Comment:
so removing this means guest can also check in themselves? supposed to be only admin/organiser/coorganiser of event only able to chck in

Reply:
@nnhathung

Reply:
@cweitat updated, only admin/organiser/coorganiser of event can do checkin